src/pagination/pagination.less

```diff
@@ -1,6 +1,9 @@
-.content-view-pf-pagination .form-group {
-  align-items: center;
-  .per-page-label {
-    padding-left: 10px;
+.content-view-pf-pagination {
+  border-top: none;
+  .form-group {
+    align-items: center;
+    .per-page-label {
+      padding-left: 10px;
+    }
```

Comment:
Shouldn't this be in Patternfly styling? Not sure why the original was here. @dtaylor113 ?

Reply:
Without the less, I get:

![image](https://user-images.githubusercontent.com/12733153/32227809-beaaeb36-be22-11e7-9ae8-0a42d563543c.png)

I'd have to dive deeper into the css used in the a-pf pagination component.  I'm fairly certain it is very close to the css used in pf core.

Reply:
Can this change be done in a separate PR so we can merge & close this one?

Reply:
Looks like the real issue here is that the pagination component should have the 'table-view-pf-pagination' selector as well as the 'content-view-pf-pagination'. If we add that for the pagination component when used for a table this change would not be necessary.